src/test/regress/expected/isolation_range_copy_vs_all.out

```diff
@@ -102,7 +102,7 @@ step s1-commit: COMMIT;
 step s1-select-count: SELECT COUNT(*) FROM range_copy;
 count
 
-10
+15
```

Comment:
Investigating why this change happened. Similar to https://github.com/citusdata/citus/pull/4207/files#r497480623